ntc_templates/templates/zyxel_os_sys_atsh.textfsm

```diff
@@ -6,7 +6,12 @@ Value SERIAL_NUMBER (\S+)
 Value MAC_ADDRESS ([0-9A-F]{12})
 
 Start
-  ^\d{4}-\d{2}-\d{2}.* -> Next
+  ^\d{4}-\d{2}-\d{2}.*
+  ^mtd\[readflash\].+
+  ^Unlocking\s.+
+  ^Reading\sfrom.+
+  ^\/tmp.+
+  ^zyUtil.+
   ^Firmware\sVersion\s*:\s${VERSION}
   ^Bootbase\sVersion\s*:\s${BOOT_VERSION}
   ^Vendor\sName\s*:\s${VENDOR}
```

Comment:
I'm suggesting to make the whitespace regex a bit more robust between words.
(So taking some of the plain `\s` and making them `\s+`.)

_(It's something we can do while we're already working on the template, or at least that's the thought.)_

Thank you for considering this and clicking the "commit suggestion" button to bring it into this PR if you choose to accept.

```suggestion
  ^Reading\s+from.+
  ^\/tmp.+
  ^zyUtil.+
  ^Firmware\s+Version\s*:\s${VERSION}
  ^Bootbase\s+Version\s*:\s${BOOT_VERSION}
  ^Vendor\s+Name\s*:\s${VENDOR}
```

Reply:
@k-ribot
Am I right in suspecting there isn't a "commit suggestions" button due to the Org when I make suggestions?

Thank you.